### PR TITLE
448 zopen clone spits out errors

### PR DESCRIPTION
**Pull request by DevonianTeuchter**
Rework clone logic, splitting zopen-init into a zopen-clone

### CODE REVIEW

File: bin/lib/zopen-clone [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+      verbose=true
+      ;;
+    "--debug") 
+      verbose=true
+      debug=true
+      ;;
+    "--xdebug")
+      set -x
+      ;;
+    "--yes" | "-y")
```
Comment (MikeFultonDev) [inline bin/lib/zopen-clone:78]: Can you add a —version in?
Note I moved stuff around so the new zopen-clone will need to be in bin. 
for consistency I have a short function at the start of all zopen scripts - ca you add that in?

File: bin/lib/zopen-clone [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+cat << HELPDOC
+zopen clone is a utility for z/OS Open Tools to generate a clone
+an existing zopen environment. For example a user can install
+to a test area and when content that the behaviour is correct, clone
+to a production area.
+
+Usage: zopen clone [OPTION] [DESTINATION]...
+
+Options:
+  -f, --from        the zopen envionrment to copy from; if not present,
```
Comment (MikeFultonDev) [inline bin/lib/zopen-clone:20]: Typo on environment

File: bin/lib/zopen-clone [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+configFile="$clonefs/etc/zopen-config"
+printInfo "- Creating configuration for cloned environment"
+
+printDebug "Removing existing configuration file"
+[ -e "$configFile" ] && rm "$configFile"
+writeConfigFile "$configFile" "$clonefs" "$zopen_pkginstall" "$ZOPEN_CA"
+
+[ ! -r $configFile ] && printError "Config file '$configFile' was not created as expected. Correct any reported errors or run with --verbose option for more details"
+
+perms="744"
```
Comment (MikeFultonDev) [inline bin/lib/zopen-clone:212]: Do we need to give ‘others’ read access?

**Review by MikeFultonDev — COMMENTED** [review]
Overall seems good! Had a couple comments. The merge might be a little annoying - sorry - we hit similar files

File: bin/lib/zopen-clone (hunk 2 above)
Comment (DevonianTeuchter) [reply bin/lib/zopen-clone]: fixed

File: bin/lib/zopen-clone (hunk 3 above)
Comment (DevonianTeuchter) [reply bin/lib/zopen-clone]: changed in latest commits

File: bin/lib/zopen-clone (hunk 1 above)
Comment (DevonianTeuchter) [reply bin/lib/zopen-clone]: done

**Comment by DevonianTeuchter** [comment]
Closing - superceded by 448-redux which removes the mess of merges!
It also takes into account @MikeFultonDev comments above so they are not lost

**Pull request closed without merging**